<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineVerify_Dialog</class>
 <widget class="QDialog" name="LineVerify_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>323</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>462</width>
    <height>323</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>462</width>
    <height>323</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Line verify</string>
  </property>
  <widget class="QLineEdit" name="_106">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>162</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_48">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>162</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Direction</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="_105">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>132</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_46">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>102</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="_104">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>102</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_47">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>132</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>End</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_52">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>262</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>XY angle</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="_108">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>232</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="_110">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>262</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_51">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>232</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Angle 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="_1">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>292</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_2">
   <property name="geometry">
    <rect>
     <x>366</x>
     <y>292</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_53">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>232</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Angle 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_54">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>262</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>XY angle 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="_109">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>232</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="_111">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>262</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_55">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>194</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Length</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_107">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>192</y>
     <width>119</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-100000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>7</y>
     <width>415</width>
     <height>87</height>
    </rect>
   </property>
   <property name="title">
    <string>Defaults</string>
   </property>
   <widget class="QPushButton" name="_1002">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>55</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select Style</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../RealCAD.qrc">
      <normaloff>:/images/PROPERTIES-styles.svg</normaloff>:/images/PROPERTIES-styles.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_103">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>56</y>
      <width>37</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Weight</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_1001">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>28</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select Layer</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../RealCAD.qrc">
      <normaloff>:/images/PROPERTIES-layers.svg</normaloff>:/images/PROPERTIES-layers.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="_102">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>56</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_100">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>31</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Colour</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_102">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>57</y>
      <width>39</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Style</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_1000">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>29</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select Colour</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../RealCAD.qrc">
      <normaloff>:/images/PROPERTIES-colours.svg</normaloff>:/images/PROPERTIES-colours.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="_100">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="WeightSpinBox" name="_103">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>54</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>3.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="LayerSpinBox" name="_101">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>28</y>
      <width>149</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="maximum">
     <number>2049</number>
    </property>
   </widget>
   <widget class="QPushButton" name="_1003">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>54</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select Weight</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../RealCAD.qrc">
      <normaloff>:/images/PROPERTIES-weights.svg</normaloff>:/images/PROPERTIES-weights.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_101">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>33</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Layer</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RCDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>../RealCADQt/source/rcdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>WeightSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>../RealCADQt/source/weightspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>LayerSpinBox</class>
   <extends>QSpinBox</extends>
   <header>../RealCADQt/source/layerspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../RealCAD.qrc"/>
 </resources>
 <connections/>
</ui>
